atlassian_jwt_auth/frameworks/common/backend.py

```diff
@@ -81,17 +81,21 @@ def get_asap_token(self, request):
 
         return auth_values[1]
 
-    def get_verifier(self):
+    def get_verifier(self, settings=None):
         """Returns a verifier for ASAP JWT tokens"""
-        settings = self.settings
+        if settings is None:
+            settings = self.settings
 
         retriever = settings.ASAP_KEY_RETRIEVER_CLASS(
             base_url=settings.ASAP_PUBLICKEY_REPOSITORY
         )
+        kwargs = {}
```

Comment:
Hmm we might be missing a test to check when the subject and issuer differ for a token - that we reject that token.

Reply:
I have added a test to cover the case I was thinking about.

Reply:
Sounds great!